test/Chainweb/Test/Pact/PactReplay.hs

```diff
@@ -12,40 +12,32 @@ import Control.Concurrent.MVar
 import Control.Monad.Catch
```

Comment:
Modernized this test for no particular reason. Hoping to get rid of `withPact` and `mkTestExecTransactions` at some point.